bin/vue-init

```diff
@@ -101,11 +101,7 @@ function run () {
       if (!hasSlash) {
         // use official templates
         template = 'vuejs-templates/' + template
-        if (template.indexOf('#') !== -1) {
-          downloadAndGenerate(template)
-        } else {
-          checkDistBranch(template, downloadAndGenerate)
-        }
+        checkDistBranch(template, downloadAndGenerate)
```

Comment:
I think there should be no changes here in `vue-init` file. We shouldn't call `checkDistBranch` for every case because with that user can't specify the branch from which she/he wants to download template for.